app/router.js

```diff
@@ -2,10 +2,13 @@ import Ember from 'ember';
 import config from './config/environment';
 
 const Router = Ember.Router.extend({
-  location: config.locationType
+  location: config.locationType,
 });
 
 Router.map(function() {
+  this.route('new');
+  this.route('detail',{ path: `/:_id` });
+  this.route('index');
```

Comment:
Don't register the `index` route or else your home page will be `localhost:4200/index` instead of `localhost:4200`